### DISCUSSION TITLE
기능별 branch 분리에 관한 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#96] [Q&A] [answered]
프로젝트 작업을 할 때 되게 체계적으로 세분화해서 Issues로 만들어서 관리중입니다.

며칠뒤 프로젝트 발표가 있어서 되게 많은 기능들에 대해 약간의 수정이 들어왔습니다.

원래는 이슈를 다시 reopen해서 브랜치로 만들어서 작업을 해왔는데 양이 UI쪽에서 세세한 부분이 많다보니 지금은 작업 날짜를 기준으로 브랜치를 만들어서 작업중인데 그렇더라도 기능을 바인딩하는게 더 좋을까요?

**Comment by jongfeel** [comment, marked as answer]
배포, 테스트용 branch가 아닌 이상 작업 branch는 기능별로 checkout을 하고 merge하는 방식이 좋습니다.
기존의 issue를 reopen 하기 보다는, 새로운 feature issue로 등록해서 하는게 좋을 것 같네요.
